recipes-kernel/perf/perf.bbappend

```diff
@@ -0,0 +1,4 @@
+# As of linux-next-20191022, we need bpf_helpers_doc.py in
+# order to build libbpf, which lives in /scripts. See
+# commit 2f184393e0c2.
```

Comment:
maybe a git subject here would help since the next hash is ephemeral. Also, this is just the resolution commit - where did the bpf_helpers_doc,py dependency come from?

Reply:
I thought the commit ID would stay the same once it lands into a maintainer's tree, through the series of git-merges until it gets into mainline. I'll add the subject nonetheless.
What I can see from my git-excavation is that the script has lived there since forever, and it's just now that the Makefile is calling it. It failed on OE because the /scripts directory was not copied over.
The merge conflict is apparently unrelated.

Reply:
Oh, I used the wrong commit. Fixing that.